pkg/util/ec2_interface.go

```diff
@@ -12,7 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package cloud
+package util
```

Comment:
I had to move this (and the other interface) here to avoid a circular dependency where the plugin depends on cloud and cloud depends on the plugin.